src/main/java/org/opensearch/flowframework/rest/RestCreateWorkflowAction.java

```diff
@@ -253,7 +253,7 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
 
             return channel -> client.execute(CreateWorkflowAction.INSTANCE, workflowRequest, ActionListener.wrap(response -> {
                 XContentBuilder builder = response.toXContent(channel.newBuilder(), ToXContent.EMPTY_PARAMS);
-                channel.sendResponse(new BytesRestResponse(RestStatus.CREATED, builder));
+                channel.sendResponse(new BytesRestResponse(RestStatus.ACCEPTED, builder));
```

Comment:
this should only be accepted if provision or reprovision is true, otherwise created